opt4j-optimizers/src/main/java/org/opt4j/optimizers/ea/Nsga2.java

```diff
@@ -184,43 +183,27 @@ public int compare(Individual o1, Individual o2) {
 	 */
 	public List<List<Individual>> fronts(Collection<Individual> individuals) {
 
-		List<Individual> pop = new ArrayList<Individual>(individuals);
-		Map<Individual, Integer> id = new HashMap<Individual, Integer>();
-		for (int i = 0; i < pop.size(); i++) {
-			id.put(pop.get(i), i);
+		List<Individual> population = new ArrayList<Individual>(individuals);
+		Map<Individual, Integer> individualID = new HashMap<Individual, Integer>();

```

Comment:
naming: better use plural for maps and sets. individualIds

Reply:
Done.